mysite/settings.py

```diff
@@ -107,3 +114,10 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+
+
+# Heroku: Update database configuration from $DATABASE_URL.
+
+# pylint: disable=invalid-name
+db_from_env = dj_database_url.config(conn_max_age=500)
+DATABASES['default'].update(db_from_env)
```

Comment:
This hackery isn't needed. You can just assign the config where you declare the databases.